complex objects/cargo/src/logins.rs

```diff
@@ -116,8 +115,16 @@ impl LoginManager {
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+#[repr(u8)]
```

Comment:
If this is going across the FFI boundary, it should be `repr(C)`; if it works with `repr(u8)` then you got lucky! It's possible that you've found more up-to-date docs than the ones I've read. If so, a pointer to those would be great.

For an enum, `repr(C)` means `repr(u*)`, with whatever size the platform would use for a C enum, but it also guarantees that it'll work correctly across the FFI. I don't think `u8` necessarily does so.

Reply:
I got that from some cbindgen examples http://dreamingofbits.com/post/generating-c-bindings-for-rust-crates-with-cbindgen/. But you're right, `repr(C)` is preferable here.